### PR TITLE
pcp-slabinfo: initial commit of PCP implementation of slabinfo

### PR DESCRIPTION
**Pull request by orasagar**

### CODE REVIEW

File: qa/archives/GNUmakefile [hunk 1]
```diff
@@ -47,7 +47,8 @@ DEFLATE = small.0.xz small.meta small.index \
 	  pcp-pidstat-process-states.index pcp-pidstat-process-args.0.xz \
 	  pcp-pidstat-process-args.index pcp-pidstat-process-args.meta.xz \
 	  pcp-pidstat-timezone.0.xz pcp-pidstat-timezone.meta.xz \
-	  pcp-pidstat-timezone.index zero_disk_activity.index  \
+	  pcp-pidstat-timezone.index pcp-slabinfo.0 pcp-slabinfo.index \
```
Comment (natoscott) [inline qa/archives/GNUmakefile:50]: Could you compress these?  Just saves a little space and time on git clone/pull for everyone ... thanks.

File: src/pcp/slabinfo/GNUmakefile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+default_pcp : default
+
+install_pcp : install
+
+check:: $(SCRIPT)
+	$(PYLINT) $^
+
+check :: $(MAN_PAGES)
+	$(MANLINT) $^
```
Comment (natoscott) [inline src/pcp/slabinfo/GNUmakefile:43]: This man lint check is showing some issue in CI - can you check that?  (see the Fedora failures, those are the only hosts with mandoc installed).  Thanks!

File: src/pcp/slabinfo/pcp-slabinfo.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        self.context=None
+
+if __name__ == '__main__':
+    try:
+        opts = SlabinfoOptions()
+        mngr = pmcc.MetricGroupManager.builder(opts,sys.argv)
+        opts.context=mngr.type
+        missing = mngr.checkMissingMetrics(ALL_METRICS)
+        if missing is not None:
+            sys.stderr.write(F"Error:Metric is {missing} missing\n")
```
Comment (natoscott) [inline src/pcp/slabinfo/pcp-slabinfo.py:192]: I think the grammar here is incorrect re "Metric is {...names-of-metrics...} missing",
perhaps something like the following would read better:
```
    sys.stderr.write("Error: some metrics are unavailable", missing)
```

**Review by natoscott — CHANGES_REQUESTED** [review]
@orasagar looking good - just a couple of small things I noticed.

**Comment by orasagar** [comment]
@natoscott  i have fixed the review comment.Please check and let me know

**Comment by orasagar** [comment]
@natoscott I have added the 1989.out test output file

File: qa/archives/GNUmakefile (hunk 1 above)
Comment (orasagar) [reply qa/archives/GNUmakefile]: Resolved with commit  "pcp-slabinfo: review comment rework"

**Pull request merged**